src/Microsoft.DotNet.Build.Tasks/GenerateEncodingTable.cs

```diff
@@ -0,0 +1,395 @@
+﻿using Microsoft.Build.Framework;
```

Comment:
Done.  I also loosened the validation back up, after reading a bit more about (at least one of) the code pages/encodings.  (We probably need to add/change a couple of entries, but that's independent of the program itself).